fix(personal-assistant): preserve ready-draft cadence narrowing

## Problem

Current `develop` fails the required root verification gate in `@elizaos/plugin-personal-assistant` after #20182 widened a parked schedule-awaiting draft so its cadence may be absent:

```text
src/actions/life.ts(4693,15): error TS2345: LifeOpsCadence | undefined is not assignable to LifeOpsCadence
src/actions/life.ts(4707,153): error TS2345: LifeOpsCadence | undefined is not assignable to LifeOpsCadence
src/actions/life.ts(4789,9): error TS2322: LifeOpsCadence | undefined is not assignable to LifeOpsCadence
```

The create handler already returns from its missing-cadence branch before constructing the save-ready draft, but an explicit `DeferredLifeDefinitionDraft` annotation widens the concrete request back to the union's optional cadence shape. Preserve the runtime guard and the narrower inferred ready-draft type while checking it against the public deferred-draft contract.

## Acceptance criteria

- [ ] The ready definition draft retains a required `LifeOpsCadence` after the missing-cadence early return.
- [ ] Schedule-awaiting drafts may still omit cadence only while `awaitingField: "schedule"`.
- [ ] Focused deferred-draft/create behavior remains green.
- [ ] Package typecheck and root `bun run verify` pass on the exact head.

AI provider/model: OpenAI / gpt-5.6-sol
Client / agent tooling: Codex Desktop
Contribution skill revision: N/A - contribute-to-eliza skill was not available in this session
Attribution status: self-reported
— [codex/lifeops-cadence-narrowing]
<!-- eliza-computer-attribution:v1 {"provider":"openai","model":"gpt-5.6-sol","client":"Codex Desktop","skill_revision":"N/A - contribute-to-eliza skill was not available in this session"} -->

--- a/plugins/plugin-personal-assistant/src/actions/life.ts
+++ b/plugins/plugin-personal-assistant/src/actions/life.ts
@@ -4619,7 +4619,7 @@ async function runLifeOperationHandlerInner(
                 multiStep: llmPlan?.multiStep === true,
               }),
             });
-      const definitionDraft: DeferredLifeDefinitionDraft = {
+      const definitionDraft = {
         intent,
         operation: "create_definition",
         createdAt: editingDeferredDefinitionDraft
@@ -4671,7 +4671,7 @@ async function runLifeOperationHandlerInner(
               | CreateLifeOpsDefinitionRequest["websiteAccess"]
               | undefined) ?? deferredDefinitionDraft?.request.websiteAccess,
         },
-      };
+      } satisfies DeferredLifeDefinitionDraft;
       if (
         shouldRequireLifeCreateConfirmation({
           confirmed: createConfirmed,

